test/export/StructureDefinitionExporter.test.ts

```diff
@@ -1434,6 +1448,28 @@ describe('StructureDefinitionExporter', () => {
       }
     ]);
   });
+  it('should apply a fixedValueRule on the child of a choice element with constrained choices', () => {
+    // Quantity is the first ancestor of Duration and Age
+
+    // * value[x] only Duration, Age
```

Comment:
Technically the comma isn't allowed in real FSH (it should be `Duration or Age`).  Not a big deal since it's in a comment though.

Reply:
BTW -- thanks for this example.  It actually causes errors in IG Publisher, which I *think* is a bug in IG Publisher.  So you helped surface that one.